docs/instrumentations/java/java.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Java Native Instrumentation"
+sidebarTitle: "Native instrumentation"
+---
+
+## Supported Versions
+
+Odigos uses the official OpenTelemetry instrumentation libraries and SDK components, thus it supports the same Java versions as this project.
+
+- Java Virtual Machine runtime versions **8** and above are supported.
```

Comment:
we need to also match the text [here](https://github.com/odigos-io/odigos/blob/abc489b82f258f070e183ed3e0d435deb2d9f772/docs/instrumentations/overview.mdx?plain=1#L14)